variables.pkr.hcl

```diff
@@ -104,6 +104,16 @@ variable "distribution_release_al2022" {
   description = "Amazon Linux 2022 distribution release."
 }
 
+variable "kernel_version_al2022" {
+  type        = string
```

Comment:
Should we merge `kernel_version_al2022 `and `kernel_version_al2022arm` into one variable `kernel_version_al2022` when we update the generate-release-vars.sh to use the kernel version and get AL2022 and AL2022 ARM AMI IDs from SSM? If it is yes, should we set default value `kernel-5.15` for `kernel_version_al2022 `in variables.pkr.hcl instead of adding new variables to the generate-release-vars.sh?

Reply:
Same comment as above - would like to have a separate discussion on whether we would like to start pinning kernel versions for ECS Optimized AMIs.